include/boost/geometry/geometries/point_xy.hpp

```diff
@@ -48,7 +48,7 @@ template<typename CoordinateType, typename CoordinateSystem = cs::cartesian>
 class point_xy : public model::point<CoordinateType, 2, CoordinateSystem>
 {
 public:
-    /// \constructor_default_no_init
+    /// Constructor_default_no_init
```

Comment:
Unrelated

Reply:
This was doxygen alias I think:
https://github.com/boostorg/geometry/blob/develop/doc/doxy/Doxyfile#L93

Reply:
> This was doxygen alias I think: https://github.com/boostorg/geometry/blob/develop/doc/doxy/Doxyfile#L93

Sorry. Thanks, it's fixed.